sympy/printing/tests/test_latex.py

```diff
@@ -149,6 +157,7 @@ def test_latex_symbols():
     assert latex(Symbol('omega') ** Symbol('beta')) == r"\omega^{\beta}"
 
 
+
```

Comment:
Oops, of course. Just a little question which I did not find answered in the "development workflow" page: do you prefer a new (trivial) commit for that or should I rather amend the previous commit and do a forced push?